core/arch/arm/tee/arch_svc.c

```diff
@@ -274,6 +278,19 @@ static void print_panic_stack(struct thread_svc_regs *regs)
 	struct abort_info ai;
 	struct thread_abort_regs ar;
 	uint32_t *pregs = (uint32_t *)regs->r1;
+	struct tee_ta_session *s;
+
+	if (tee_ta_get_current_session(&s))
+		panic("No current session");
+
+	if (tee_mmu_check_access_rights(to_user_ta_ctx(s->ctx),
+					TEE_MEMORY_ACCESS_READ |
+					TEE_MEMORY_ACCESS_WRITE,
+					(uaddr_t)pregs,
+					TA_CONTEXT_MAX_SIZE)) {
+		EMSG("Invalid register dump, cannot print stack");
```

Comment:
`TAMSG_RAW` for consistency.
Also, print pointer? `"Invalid user-mode pointer %0x" PRIxUA ",cannot print stack", pregs`